doc/en/source/ngsildbasediotintegration.rst

```diff
@@ -220,30 +220,7 @@ communications protocol to be used.
 	}'
 
 
-**Step 3** To see the state of the water sprinkler change through device monitor URL:**<IoT-DeviceIP>:3000/device/monitor** send the below PATCH request directly to the IoT Agent's North Port
-
-.. code-block:: console 
-
-	curl -L -X PATCH 'http://<IoT-AgentIP>:4041/ngsi-ld/v1/entities/urn:ngsi-ld:Device:water001/attrs/on' \
-    	-H 'fiware-service: openiot' \
-    	-H 'fiware-servicepath: /' \
-    	-H 'Content-Type: application/json' \
-	--data-raw '{
-
-        	"type": "Property",
-        	"value": " "
-
-	}'
-
-
-
-**step 4** To see the status of entity **urn:ngsi-ld:Device:water001** open the device dashboard in your web browser by using URL: **<IoT-DeviceIP>:3000/device/monitor** . The status should be "on".
-
-.. figure:: figures/status1.png
-
-
-
-**step 5** To verify the result from orion broker send the following request
+**step 3** After Provisioning an Actuator on IoT agent the IoT agent record a measurement of Actuator on orion-broker. To retrive the recorded measurement of actuater from context broker execute the following command. 
```

Comment:
correct the step "IoT Agent records the measurement of Actuator after Provisioning the Actuator on it  and forward the measurement to Orion. Execute the following command to retrieve the recorded measurement of actuator from Orion"

Reply:
Done